x/paloma/keeper/keeper_integration_test.go

```diff
@@ -56,7 +53,8 @@ var _ = Describe("jailing validators with missing external chain infos", func()
 			// val[1] has only one chain
 			// val[2] doesn't have anything
 			// All other vals have everything
-			err := a.ValsetKeeper.AddExternalChainInfo(ctx, vals[0].GetOperator(), []*valsettypes.ExternalChainInfo{
+			valAddress, err := k.MustGetValAddr(vals[0].GetOperator())
```

Comment:
Does this even compile? `MustGetValAddr` doesn't return two types. It's also intended as a drop in replacement for `vals[0].GetValidator()` to reduce amount of boilerplate.

Reply:
no, there is not error returned from `MustGetValAddr`. Will fix and update this once we conclude on that util function as I mentioned [above](https://github.com/palomachain/paloma/pull/1060#discussion_r1437783431)